.github/workflows/publish.yml

```diff
@@ -64,6 +65,14 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL_PYTHON_SDK }}
           SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
 
+      - name: create PR to merge release branch and master
+        run: |
+          git checkout release/${{ github.ref_name }}
+          git checkout -b merge_release/${{ github.ref_name }}
+          git merge master
```

Comment:
how to handle a case we we would have continued to develop on master and that it creates merge conflicts ?
Could it notify us on slack that we need to resolve conflict to merge the release branch into master ?